.changeset/long-gifts-protect.md

```diff
@@ -0,0 +1,31 @@
+---
+'braid-design-system': minor
+---
+
+---
+new:
+  - themes
+---
+
+Add `seekJobs` theme
+
+The `seekJobs` theme encapsulates the system changes necessary to apply and deliver the updated visual design language for SEEK Jobs.
+Through the development of this theme, we have been able improve the fidelity of the various scales in our tokens, while also ensuring that the tokens themselves are consumed and applied more consistently throughout the system itself.
+
+**EXAMPLE USAGE:**
+```tsx
+import seekJobs from 'braid-design-system/themes/seekJobs';
+
+<BraidProvider theme={seekjobs}>
+  ...
+</BraidProvider>
+```
+
+**MIGRATION**
+
+Consumers of the apac theme are not recommended to migrate independently. The seekJobs theme represents an uplifted visual identity that is part of a wider visual uplift.
```

Comment:
```suggestion
Consumers of the apac theme are not recommended to migrate independently. The `seekJobs` theme represents an uplifted visual identity that is part of a wider visual uplift.
```

Reply:
Taking the above suggestion which covers both theme references.